### ISSUE TITLE
DE_SUBJECT_VERB_AGREEMENT: False negatives

### ISSUE THREAD

**Issue by Discostu36** [languagetool-org/languagetool#6019]
These subject verb agreement errors do not get caught:

> Zum 1. Januar **treten Gesetzesänderung** zum Mindestlohn und Spitzensteuersatz in Kraft. 

> Letzten Monat **gingen die Arbeitslosenzahl** deutlich zurück.

**Comment by udomai** [comment]
Hi @Discostu36, thanks for reporting on those two cases!

The second case is a clear FN. In principle, this is a friendly context with only one finite verb in the sentence, we'll look into that.

The first case, I don't agree with, though: The subject is "(die) Gesetzesänderung zum Mindestlohn und (der) Spitzensteuersatz", thus the verb must be plural. Amirite?

**Comment by Discostu36** [comment]
> The first case, I don't agree with, though: The subject is "(die) Gesetzesänderung zum Mindestlohn und (der) Spitzensteuersatz", thus the verb must be plural. Amirite?

No, only "Gesetzesänderungen" is subject, "zum Mindestlohn und Spitzensteuersatz" is object (or maybe adverbial? Not sure). But we can make it more simple, the problem is the same

> Zum 1. Januar **treten Gesetzesänderung** in Kraft.

**Comment by Discostu36** [comment]
Another example:

> Mein Ziel ist es, dass sich die Kunden bestens aufgehoben und beraten fühlt.
